### PR TITLE
tmp: Log out state when redirecting e2e atomic test site

### PR DESCRIPTION
**Pull request by WunderBart**
> **Warning**
> DO NOT MERGE

This is a temporary change that should help us understand why the auth loop happens (#69865).

### CODE REVIEW

File: client/gutenberg/editor/controller.js [hunk 1]
```diff
@@ -186,6 +187,17 @@ export const authenticate = ( context, next ) => {
...
+	if ( siteId === 204039232 ) {
+		logToLogstash( {
+			feature: 'calypso_client',
+			message: 'e2e atomic auth redirect',
+			severity: 'debug',
+			extra: {
+				state: JSON.stringify( state, null, 2 ),
+			},
+		} );
+	}
```
Comment (tyxla) [inline client/gutenberg/editor/controller.js]: Let's add a comment before shipping. `TODO: Remove me ASAP` or something does the job.

**Review by tyxla — APPROVED** [review]
Let's add a comment to make sure to follow up and clean it up shortly.

File: client/gutenberg/editor/controller.js (hunk 1 above)
Comment (WunderBart) [reply client/gutenberg/editor/controller.js]: It looks like it won't be necessary to merge it, so I'll just close the PR when ready. Does that sound OK?

Comment (tyxla) [reply client/gutenberg/editor/controller.js]: Cool! Sounds good!

**Pull request closed without merging**